doc/installation/hpc_install.rst

```diff
@@ -171,17 +171,19 @@ On the K Computer:
       make -j4
       make install
 
-  The compilation can take quite some time compiling the file ``models/modelsmodule.cpp``
-  due to generation of many template classes. To speed up the process, you can
-  comment out all synapse models you do not need.
-  The option ``-Kfast`` on the K computer enables many different options::
+  The compilation of the built-in models can take quite some time due
```

Comment:
I suppose this was exclusively relevant for K, which has been decomissioned. Should be replaced with a section for Fugaku, shouldn't it? But that (and BG/Q above) are rather topics for a separate issue.

Reply:
I agree that this information is pretty outdated (cf. the removal of INSTALL in the top directory). I still would rather refer a decision about this to @sarakonradi and the Docteam, as it is not really in the scope of this PR.